cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityTopicOperatorTest.java

```diff
@@ -303,4 +330,87 @@ public void testRoleBindingInTheSameNamespace()   {
         assertThat(binding.getRoleRef().getKind(), is("Role"));
         assertThat(binding.getRoleRef().getName(), is("foo-entity-operator"));
     }
+
+    @ParallelTest
+    public void testFromCrdWithCruiseControl() {
```

Comment:
Not sure this is really testing `fromCrd`. It is testing that the cruise control setup gives you the right env vars and right volume names. The test is fine, but maybe you can give it a better name?

Reply:
Renamed.